examples/bare_etiss_processor/base.ini

```diff
@@ -1,29 +1,29 @@
 [StringConfigurations]
 
-ETISS::outputPathPrefix=
+etiss.output_path_prefix=
 
 
 [BoolConfigurations]
 
 norangeexception=false
 returnjump=false
 hostendianness=false
```

Comment:
Right now only the configurations that can be set directly from the command line have been renamed appropriately. Do you suggest to change the naming conventions of the other configurations set via .ini files too? Since these are displayed when parsed through, it now looks very uneven on the output window. I can change these as well if I get a proper description of what they do, so that I can define the scope correctly.

Reply:
Going through these in the slides, it looks like these are the ones that were candidates for removal.

Let's go for this:
- norangeexception: remove
- returnjump: remove
- hostendianess: remove
- copy-headers: remove
- sigint-console: remove
- library-loading: remove (used in code, but must be true for etiss to function)
- integrated-library: kinda required for etiss to function, but there might be use cases to disable it. rename to: "etiss.load_integrated_libraries" with default value true
- pyconsole: remove
- ETISS::xxx, vp::sim_time: keep them like this for now, they still need some discussion as it is unclear how they are used

Reply:
This is done. I have also removed some calls depending on these configs which never get executed. Please take a look and let me know if it's okay.